analysis.ml

```diff
@@ -1,34 +1,44 @@
 open Instr
 
-let successors (instrs : instruction_stream) pc =
+let pcs (instrs : instruction_stream) : pc array =
+  Array.mapi (fun pc _ -> pc) instrs
```

Comment:
the pc still is 0...(N-1). This helper just creates an increasing integer array the same size as the instruction stream. It's there since I did not find a convenient ocaml one-liner for ```(0...N)```. Do you know one?